admin/rpc/updateInstallation.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include_once('../../config/symbini.php');
+include_once($SERVER_ROOT . '/classes/PortalIndex.php');
+header('Content-Type: text/html; charset=' . $CHARSET);
+
+$portalID = filter_var($_REQUEST['portalID'], FILTER_SANITIZE_NUMBER_INT);
```

Comment:
Just confirming that the request would never accommodate multiple portal IDs at once, right? Otherwise, this sanitize would break it.